lte/gateway/c/oai/lib/openflow/controller/ControllerEvents.cpp

```diff
@@ -89,10 +89,26 @@ void ExternalEvent::set_of_connection(fluid_base::OFConnection* ofconn) {
   ofconn_ = ofconn;
 }
 
+UeNetworkInfo::UeNetworkInfo(const struct in_addr ue_ip)
+    : ue_ip_(ue_ip),
+      vlan_(-1) {}
+
+UeNetworkInfo::UeNetworkInfo(int vlan, const struct in_addr ue_ip)
```

Comment:
put `vlan` after `ue_ip`

Reply:
it does not look right to me, IP is in vlan, so vlan shld be first parameter.